tests/unit/ClientTest.php

```diff
@@ -174,9 +182,9 @@ protected function doTestResponse($responseFile, array $result, $method)
      *
      * @return Client
      */
-    protected function getClient($username = '', $password = '',  $ssl = false, $timeout = 0)
```

Comment:
I was not aware that BC was an issue with the Test classes.
I don't mind changing it, but want to make sure it's really necessary first.